opossum-tool/OpossumUI
# feat: offer package search suggestions

### Summary of changes

- Use react query to load search suggestions from deps.dev
- Create faker module for mocking package search hooks

### Context and reason for change

A step toward #2394 

### How can the changes be tested

Start the app and open the file opossum_input.opossum from the example-files folder.
In Audit View, choose the root folder. In the Network tab of the DevTools, check that there is an entry with name "suggest?q=Vue". Check that the respone for this entry contains the desired search results.
